lib/validators.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const Joi = require('joi');
+
+const errors = require('./errors');
+
+const validators = {schemas: {}};
```

Comment:
can this schema just be placed in `auth-client.js`? I don't see the need for putting it in a separate file

Reply:
So, I just realized we may want to extend our validation to ```getAuthUrl``` in the future. It takes in a request object just like this. If that's the case, it would make sense to keep a new file. But, if that's not important right now, I can move to ```auth-client.js```.

Reply:
Even if we extend this to `getAuthUrl` imo the schema should be in the method that is being validated. This case should almost never get hit during production and will most likely get hit while the developer is testing/building their app, so the schema will almost serve as another form of documentation for the code and I personally would like to be as close to the method as possible.